packages/ag-charts-enterprise/src/features/color-picker/colorPickerStyles.css

```diff
@@ -24,6 +16,12 @@
     --checker: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="4" height="4"><rect x="0" y="0" width="4" height="4" fill="%23fff"/><path d="M0 0H2V4H4V2H0Z" fill="%23b2b2b2"/></svg>');
 }
 
+.ag-charts-color-picker__content {
```

Comment:
Did the layout change here?

Reply:
Yeah, it's been flattened a little, merging `.ag-charts-popover` and `.ag-charts-color-picker` onto the same element (removing the `colorPickerContainer` element which served the same purpose as the outer popover element).

Ideally the `__content` styles could be merged into `.ag-charts-color-picker` but I had some issues with the width of the sliders. Will double check that.

From:
<img width="436" alt="Screenshot 2024-08-15 at 16 47 57" src="https://github.com/user-attachments/assets/74bb26ee-f800-4713-95da-2a468f10c54a">

To:
<img width="662" alt="Screenshot 2024-08-15 at 16 48 24" src="https://github.com/user-attachments/assets/f5e51089-d87f-49db-b4a5-8472842c4970">